### ISSUE TITLE
to use a saved profile in the command line parameters, must use quotation marks

### ISSUE THREAD

**Issue by ElusiveT** [mausimus/ShaderGlass#101]
maybe everybody else knows this but it took me a while to figure it out. 

in order to load a custom profile on launch, you need to put the file in the install folder and the profile filename in quotes. it works fine. there's no actual issue with the software. i just got confused because the readme doesn't specify how to do it.

it works like this in command line:
`shaderglass.exe "profile 01.sgp"`

that's all. submitting this issue in case it helps anyone else who got stuck here

**Comment by mausimus** [comment]
Yes that's a standard requirement on command line but I will add to documentation.

**Comment by mausimus** [comment]
Added to README
